requirements.dev.txt

```diff
@@ -2,36 +2,64 @@
 # This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
-#    pip-compile --no-annotate --output-file=requirements.dev.txt requirements.dev.in
+#    pip-compile --no-annotate --output-file=requirements.dev.txt requirements.dev.in requirements.in
 #
 appdirs==1.4.4
 attrs==21.4.0
 black==21.7b0
+blinker==1.4
+cachetools==4.2.4
```

Comment:
The diff of this file should mainly have the addition of the pinned versions from production.